### PR TITLE
Bug 1137547 - [RTL][Search] The dropdown arrow is on the wrong side next...

### PR DESCRIPTION
**Pull request by gmarty**
... to the search provider

### CODE REVIEW

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7b0754566e45)

File: apps/search/style/search.css [hunk 1]
```diff
@@ -291,3 +290,12 @@ gaia-grid {
 [data-loading=true] .loading {
   display: block;
 }
+
+html[dir="rtl"] #suggestions-provider {
+  padding: 0 1rem 0 2rem;
```
Comment (KevinGrandon) [inline apps/search/style/search.css]: I believe the convention for padding has been to use moz-padding-start/moz-padding-end, no?

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d5374c4a2246)

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1179a25afd62)

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f2043c58dff2)

File: apps/search/style/search.css (hunk 1 above)
Comment (gmarty) [inline apps/search/style/search.css]: I tried to remove the `direction: ltr;` in https://github.com/mozilla-b2g/gaia/pull/28630/files#diff-f5140311864b481688975f876b111d48L23, but I bump into a platform bug where the #suggestions-provider element reports no width when in RTL, so I can't place the arrow correctly.
I'm working on a reduced test case for this bug, but in the meantime we have to do without `-moz-padding-(start|end)`.

Comment (KevinGrandon) [inline apps/search/style/search.css]: I see. Thanks for the investigation. It might be worth a comment pointing to that bug.

Comment (gmarty) [inline apps/search/style/search.css]: Note added!

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7b5b2c2739a7)

**Pull request merged**